Content.Client/AME/AMEControllerVisualizerSystem.cs

```diff
@@ -0,0 +1,60 @@
+using Content.Client.AME.Components;
+using Robust.Client.GameObjects;
+using static Content.Shared.AME.SharedAMEControllerComponent;
+
+namespace Content.Client.AME;
+
+public sealed class AMEControllerVisualizerSystem : VisualizerSystem<AMEControllerVisualsComponent>
+{
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<AMEControllerVisualsComponent, ComponentInit>(OnComponentInit);
+    }
+
+    private void OnComponentInit(EntityUid uid, AMEControllerVisualsComponent component, ComponentInit args)
+    {
+        if(EntityManager.TryGetComponent<SpriteComponent>(uid, out var sprite)) {
+            sprite.LayerMapSet(AMEControllerVisualLayers.Display, sprite.AddLayerState("control_on"));
+            sprite.LayerSetVisible(AMEControllerVisualLayers.Display, false);
+        }
```

Comment:
```suggestion
        if(TryComp<SpriteComponent>(uid, out var sprite)) 
        {
            sprite.LayerMapSet(AMEControllerVisualLayers.Display, sprite.AddLayerState("control_on"));
            sprite.LayerSetVisible(AMEControllerVisualLayers.Display, false);
        }
```

codebase convention is C#-style brackets rather than egyptian brackets. same goes for everywhere else where I'll note it

Reply:
`TryComp` is just a shorter, `EntitySystem`-only version of the same thing